source/physics_lists/PetaloPhysics.cc

```diff
@@ -93,4 +102,28 @@ void PetaloPhysics::ConstructProcess()
     pmanager->RemoveProcess(cs);
   }
 
+
+  if (nest_) {
+    PetaloDetector* petalo = new PetaloDetector();
+    G4double e_field = petalo->FitEF(0, 0, 0);
```

Comment:
If you go to the NEST code, you'll see that it is the position where the electric field is evaluated. In our case, since it is constant, you could use whatever value.